examples/Quadrotor/office.urdf

```diff
@@ -384,37 +389,111 @@
     </collision>
    </link>
 
-
    <joint name="joint1" type="fixed">
    	<parent link="wall1"/>
         <child link="wall2"/>
+<<<<<<< HEAD
+=======
 <!--        <child link="TableLegUL"/>
         <child link="TableLegUR"/>
         <child link="TableLegLR"/>
         <child link="TableLegLL"/>
         <child link="TableTop"/>
         <child link="cabinet"/> -->
+>>>>>>> b667715aab7a2c38a0358cdbc5efd56d4cb191f6
```

Comment:
You've got some conflict markers committed in this file. Can you clear these up?